### DISCUSSION TITLE
Apache Age example

### DISCUSSION THREAD

**Discussion by deka** [PostgREST/postgrest#2192] [Q&A] [answered]
Hi,
Does anyone have an example of integrating the [Apache Age](https://age.apache.org/) extension with Postgrest ?
Thanks

**Comment by wolfgangwalther** [comment, marked as answer]
@deka did you try it and have anything to share yourself by now?

**Reply by deka** [reply]
Hi @wolfgangwalther ,
For now, i  use [Neo4j](https://neo4j.com/download-neo4j-now/) to poc.
Regards
